test/js/specs/happy.js

```diff
@@ -2,125 +2,454 @@ const path = require('path')
+      documentUploadConfirmation.errorTitleIcon.isDisplayed()
+      const errorInstruction = documentUploadConfirmation.errorInstruction.getText()
+      expect(errorInstruction).to.equal(documentUploadConfirmationLocale["errors"]["invalid_capture"]["instruction"])
+      documentUploadConfirmation.errorInstruction.isDisplayed()
+    })
+
+    it('should upload a document after retrying', async () => {
+      driver.get(localhostUrl)
+      welcome.primaryBtn.click()
+      documentSelection.passportIcon.click()
+      const inputImg = documentUpload.getUploadInput()
+      inputImg.sendKeys(path.join(__dirname, '../../features/helpers/resources/llama.pdf'))
+      waitForUploadToFinish
+      documentUploadConfirmation.confirmBtn.click()
+      waitForUploadToFinish
+      documentUploadConfirmation.redoBtn.click()
+      const input = documentUpload.getUploadInput()
+      input.sendKeys(path.join(__dirname, '../../features/helpers/resources/passport.jpg'))
+      waitForUploadToFinish
+      const checkReadabilityText = documentUpload.title.getText()
+      expect(checkReadabilityText).to.equal(documentUploadLocale["confirm"]["document"]["title"])
+    })
+
+    it('should return file size too large message for doc', async () => {
+      driver.get(localhostUrl)
+      welcome.primaryBtn.click()
+      documentSelection.passportIcon.click()
+      const input = documentUpload.getUploadInput()
+      input.sendKeys(path.join(__dirname, '../../features/helpers/resources/over_10mb_face.jpg'))
+      const uploaderError = documentUpload.uploaderError.getText()
+      expect(uploaderError).to.equal(documentUploadLocale["errors"]["invalid_size"]["message"] + ". " + documentUploadLocale["errors"]["invalid_size"]["instruction"] + ".")
+      documentUpload.uploaderError.isDisplayed()
+    })
+
+    // it('should return file size too large message for selfie', async () => {
+    //   driver.get(localhostUrl + `?async=false&language=&useWebcam=false`)
+    //   welcome.primaryBtn.click()
+    //   documentSelection.passportIcon.click()
+    //   const input = documentUpload.getUploadInput()
+    //   input.sendKeys(path.join(__dirname, '../../features/helpers/resources/passport.jpg'))
+    //   waitForUploadToFinish
+    //   documentUploadConfirmation.confirmBtn.click()
+    //   const inputSelfie = documentUpload.getUploadInput()
+    //   inputSelfie.sendKeys(path.join(__dirname, '../../features/helpers/resources/over_10mb_face.jpg'))
+    //   const uploaderError = documentUpload.uploaderError.getText()
+    //   expect(uploaderError).to.equal(documentUploadLocale["errors"]["invalid_size"]["message"] + ". " + documentUploadLocale["errors"]["invalid_size"]["instruction"] + ".")
+    //   documentUpload.uploaderError.isDisplayed()
+    // })
+
+    it('should return use another file type message', async () => {
+      driver.get(localhostUrl)
+      welcome.primaryBtn.click()
+      documentSelection.passportIcon.click()
```

Comment:
I think these 3 steps above could be abstracted it to `goToPassportScreen`
This will make it easier to refactor the tests when we are able to init the SDK on the screens we want